Eastern-Research-Group/csb-rebate-forms-app
# Feature/update nces data

## Related Issues:
* CSBAPP-229

## Main Changes:
* Update generated NCES JSON to correct missing leading zero for some zip codes (update to #345)

## Steps To Test:
1. Navigate to a 2023 submission within the app. The 2023 FRF's NCES fields should populate when a NCES ID has been entered, and specifically – you should now see the zip code include a leading zero for zip codes that begin with zero.
